bundles/org.openhab.binding.nibeheatpump/contrib/NibeGW/Arduino/NibeGW/Config.h

```diff
@@ -0,0 +1,220 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Author: pauli.anttila@gmail.com
+ *
+ */
+#ifndef Config_h
+#define Config_h
+
+// ######### BOARD SELECTION #######################
+
+// Enable if you use ProDiNo NetBoard V2.1 board
+//#define PRODINO_BOARD
+
+// Enable if you use PRODINo ESP32 Ethernet v1 (Enable also HARDWARE_SERIAL_WITH_PINS in NibeGW.h)
+#define PRODINO_BOARD_ESP32
+
+// Enable if ENC28J60 LAN module is used
+//#define TRANSPORT_ETH_ENC28J60
+
+// ######### CONFIGURATION #######################
+
+// Enable dynamic configuration mode via WiFi connection (supported only by the PRODINO_BOARD_ESP32 board)
+// Dynamic configuration mode is loaded if input 0 is ON during boot
+// When dynamic configuration mode is activated, login to the 'Bleeper' WiFi Access point:
+//  1. Configuration page is available on IP 192.168.4.1 port 80 (http://192.168.4.1).
+//  2. OTA update page is available on IP 192.168.4.1 port 8080 (http://192.168.4.1:8080/update).
+// Install following libraries via Arduino IDE library manager:
+//  1. Bleeper (tested with version 1.1.0)
+//  2. ElegantOTA (tested with version 2.2.9)
+#define ENABLE_DYNAMIC_CONFIG
+
+// Enable debug printouts.
+#define ENABLE_DEBUG
+#define VERBOSE_LEVEL           1
+#define ENABLE_SERIAL_DEBUG
+#define ENABLE_REMOTE_DEBUG     // Remote debug is available in telnet port 23
+
+#define BOARD_NAME              "Arduino NibeGW"
+
+// Ethernet configuration
+#define BOARD_MAC               "DE:AD:BE:EF:FE:ED"
+#define BOARD_IP                "192.168.1.100"
+#define DNS_SERVER              "192.168.1.1"
+#define GATEWAY_IP              "192.168.1.1"
+#define NETWORK_MASK            "255.255.255.0"
+
+// UDP ports for incoming messages
+#define INCOMING_PORT_READCMDS  9999
+#define INCOMING_PORT_WRITECMDS 10000
+
+// Target IP address and port where Nibe UDP packets are send
+#define TARGET_IP               "192.168.1.101"
+#define TARGET_PORT             9999
+
+// Delay before initialize ethernet on startup in seconds
+#define ETH_INIT_DELAY          5
+
+// Send acknowledge PDU's to Nibe
+#define SEND_ACK                true
+
+// Ack following periperial messages
+#define ACK_MODBUS40            true
+#define ACK_SMS40               false
+#define ACK_RMU40               false
+
+
+
+// Used serial port and direction change pin for RS-485 port
+#if defined(PRODINO_BOARD)
+  #define RS485_PORT            Serial1
```

Comment:
General observation: In existing example code preprocessor directives were not indented. I'm fine either way, just noticed this difference.